pages/common/fls.md

```diff
@@ -0,0 +1,16 @@
+# fls
+
+> List files and directories in an image file or device.
+> More information: <https://wiki.sleuthkit.org/index.php?title=Fls>.
+
+- Build a recursive fls list over a device:
+
+`fls -r -m {{original_path}} {{/dev/examplepartition}}`
+
+- Build a recursive fls list over a partition starting at a specific sector using a pre-defined timezone:
```

Comment:
```suggestion
- Build a recursive fls list over a partition, starting at a specific sector and using a pre-defined timezone:
```

This definitely needs to be re-worded, as the sentence is quite clumsy.

Reply:
Thanks

Reply:
I'd wait for the opinion of others on this, as I'm unsure what's the best way to phrase it.

Reply:
Would you prefer:

> Analyse a single partition, providing the timezone of the original system and the sector offset where the file system starts in the image.

Or:
> Provide the timezone of the original system and the sector offset where the file system starts in the image.

Reply:
The first one sounds more informative. Let's wait for the opinion of others.

Reply:
That first one sounds best, but perhaps a tweak?

> - Analyse a single partition, providing the timezone of the original system and the sector offset at which the file system starts in the image: